### PR TITLE
Sov 808 d 2 transaction step component

### PR DESCRIPTION
**Pull request by Rickk137**
https://sovryn.atlassian.net/browse/SOV-808

### CODE REVIEW

File: apps/frontend/src/app/3_organisms/components/TransactionGas/TransactionGas.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <AmountInput
+        label="Gas price"
+        unit="Gwei"
+        className="mb-4 max-w-64"
+        min={0}
+        value={price}
+      />
+    </>
+  );
+};
```
Comment (soulBit) [inline apps/frontend/src/app/3_organisms/components/TransactionGas/TransactionGas.tsx]:
```suggestion
export const TransactionGas: FC<TransactionGasProps> = ({ limit, price }) => (
    <>
      <AmountInput
        label="Gas limit"
        className="mt-2 mb-4 max-w-64"
        min={0}
        value={limit}
      />
      <AmountInput
        label="Gas price"
        unit="Gwei"
        className="mb-4 max-w-64"
        min={0}
        value={price}
      />
    </>
  );
```

File: packages/ui/src/2_molecules/RadioButton/RadioButton.tsx [hunk 2]
```diff
@@ -7,25 +7,38 @@ import { IRadioButtonProps } from './RadioButton.types';
...
+    <>
+      <div
+        className={classNames(styles.radioButton, disabled && styles.disabled)}
+      >
+        <label htmlFor={id} className={classNames(styles.label, className)}>
+          <input
+            id={id}
+            type="radio"
+            disabled={disabled}
+            data-layout-id={dataLayoutId}
```
Comment (soulBit) [inline packages/ui/src/2_molecules/RadioButton/RadioButton.tsx]:
```suggestion
            {...applyDataAttr(dataLayoutId)}
```
Switch this to use new utility function for applying layout id attribute, examples can be found in `SimpleTableRow` or other components

File: apps/frontend/src/app/3_organisms/components/TransactionGas/TransactionGas.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+
+import { AmountInput } from '@sovryn/ui';
```
Comment (soulBit) [inline apps/frontend/src/app/3_organisms/components/TransactionGas/TransactionGas.tsx]: Move `apps/frontend/src/app/3_organisms/components` folder to `apps/frontend/src/app/3_organisms/TransactionStep/components` and update any imports

**Review by soulBit — CHANGES_REQUESTED** [review]
Couple of changes to make from scope of this story.

There's also some functional/behavioural things not covered by this implementation (e.g. Tx id and Accordion visibility behaviour), but these can be picked up as part of SOV-823 (I'll add them to Jira)

**Review by soulBit — APPROVED** [review]

**Review by pietro-maximoff — APPROVED** [review]

**Pull request merged**